src/main.cpp

```diff
@@ -542,7 +542,7 @@ Use Ctrl+C to stop.)");
 #ifdef HAVE_PULSE
     printf(R"(
 
-  -a, --audio [DEVICE]      Starts recording the screen with audio.
+  -a, --audio[DEVICE]       Starts recording the screen with audio.
```

Comment:
Eh, actually, I think there should be a space for the long option, shouldn't it?

Reply:
That doesn't work at all:
```
$ wf-recorder --audioalsa_output.pci-0000_00_1f.3.analog-stereo.monitor -f verunka-test.mkv
wf-recorder: unrecognized option '--audioalsa_output.pci-0000_00_1f.3.analog-stereo.monitor'
Unsupported command line argument (null)
Please select an output from the list to capture (enter output no.):
1. Name: eDP-1 Description: Chimei Innolux Corporation 0x14C9 0x00000000 (eDP-1)
2. Name: HDMI-A-2 Description: Philips Consumer Electronics Company PHL 325E1 0x00003888 (HDMI-A-2)
Enter output no.:^C

$ wf-recorder --audio alsa_output.pci-0000_00_1f.3.analog-stereo.monitor -f verunka-test.mkv
Please select an output from the list to capture (enter output no.):
1. Name: eDP-1 Description: Chimei Innolux Corporation 0x14C9 0x00000000 (eDP-1)
2. Name: HDMI-A-2 Description: Philips Consumer Electronics Company PHL 325E1 0x00003888 (HDMI-A-2)
Enter output no.:2
selected region 0 0 0 0
Setting codec option: crf=20
Setting codec option: preset=ultrafast
Setting codec option: tune=zerolatency
Choosing pixel format yuv420p
[libx264 @ 0x7f9d98002700] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX FMA3 BMI2 AVX2
[libx264 @ 0x7f9d98002700] profile Constrained Baseline, level 5.1, 4:2:0, 8-bit
[libx264 @ 0x7f9d98002700] 264 - core 161 r3020 d198931 - H.264/MPEG-4 AVC codec - Copyleft 2003-2020 - http://www.videolan.org/x264.html - options: cabac=0 ref=1 deblock=0:0:0 analyse=0:0 me=dia subme=0 psy=1 psy_rd=1.00:0.00 mixed_ref=0 me_range=16 chroma_me=1 trellis=0 8x8dct=0 cqm=0 deadzone=21,11 fast_pskip=1 chroma_qp_offset=0 threads=8 lookahead_threads=8 sliced_threads=1 slices=8 nr=0 decimate=1 interlaced=0 bluray_compat=0 constrained_intra=0 bframes=0 weightp=0 keyint=250 keyint_min=25 scenecut=0 intra_refresh=0 rc=crf mbtree=0 crf=20.0 qcomp=0.60 qpmin=0 qpmax=69 qpstep=4 ip_ratio=1.40 aq=0
Output #0, matroska, to 'verunka-test.mkv':
    Stream #0:0: Unknown: none (libx264)
    Stream #0:1: Unknown: none (aac)
[matroska @ 0x7f9d98000d40] Using AVStream.codec to pass codec parameters to muxers is deprecated, use AVStream.codecpar instead.
[matroska @ 0x7f9d98000d40] Using AVStream.codec to pass codec parameters to muxers is deprecated, use AVStream.codecpar instead.
Using PulseAudio device: default
```